api/src/lib/lambda.ts

```diff
@@ -0,0 +1,6 @@
+import middy from '@middy/core'
```

Comment:
I don't think I'll continue using middy, since I author a competing middleware framework -- but it's the one their template provides and this is really just dropping the template in.